src/Lecture1.hs

```diff
@@ -119,4 +142,20 @@ and lower than 6 elements (4, 5, 6, 7, 8 and 9).
 
 🕯 HINT: Use recursion to implement this function.
 -}
-lowerAndGreater n list = error "TODO"
+lowerAndGreater :: Integer -> [Integer] -> String
+lowerAndGreater n list = show n ++ " is greater than " ++ show (greater 0 list) ++ " elements and lower than " ++ show (lower 0 list) ++ " elements"
```

Comment:
That's correct! 💯 

However, with this solution, you will walk through the list twice.
You can write it more performant with only one traversal of the list 🙂
It's usually preferred for functions to traverse lists only once and consume each element only once. So the composition of such functions works better with streaming and lazy evaluation.